IFC4x3/Constants/p/POSITION_0A_EeaX8fClRhysKUHwrUQ.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="POSITION_0A_EeaX8fClRhysKUHwrUQ" Name="POSITION" UniqueId="0af8ea24-848a-4cbd-bafc-d94791eb579a">
+	<Documentation>Used to fully describe a linearly referenced location given by the linear element being measured (the _IfcAlignment_ into which the _IfcReferent_ is nested), the method of measurement (_Pset_LinearReferencingMethod_) and a measure value specified with a distance expression (_Pset_DistanceExpression_). If a linear referencing method is specified for the position, it overrides any linear referencing method specified for the alignment.”</Documentation>
+</DocConstant>
+
```

Comment:
This one needs attention:
- Either remove it completely 
- or add a Pset_PositionExpression and change documentation or if Pset_DistanceExpression can be used (including station, lateral- and vertical offsets)